.travis.yml

```diff
@@ -3,21 +3,19 @@ services:
 - docker
 env:
   global:
-  - DIST_REPO="f5-sphinx-theme"
-  - USER_REPO=$(echo $TRAVIS_REPO_SLUG | cut -f -1 -d / )
-  - S3_DIST_URL="http://d3s2z99mfhr69y.cloudfront.net"
-  - BUILD_HTML_DIR=${TRAVIS_BUILD_DIR}/test-deploy/docs/_build/html
-  - BRANCH_DIR=${DIST_REPO}/travis-builds/${USER_REPO}/${TRAVIS_BRANCH}/
   - AWS_S3_BUCKET=${ARTIFACTS_BUCKET}
-  - UPLOAD_DIR="f5-sphinx-theme/$(date +%F)/${TRAVIS_BUILD_NUMBER}"
+  - USER_REPO=$(echo $TRAVIS_REPO_SLUG | cut -f -1 -d / )
+  - PROJECT_UPLOAD_ROOT="under-review/f5-sphinx-theme"
```

Comment:
This is not the correct path for this repo -- it doesn't need the "under-review".

Reply:
The only reason I have this is that it allows me to run a build from a fork with the AWS credentials I have from my docs project. If you can set me up with a token that has access to the build folder (or give the iapps lx docs account access) then this would not be needed.

Reply:
The content in this repo actually deploys to a different bucket than the docs under review. I'm not clear on how that impacts the iapplx-docs project, but we can discuss offline.